### DISCUSSION TITLE
Need a option to hide starred repositories from public view.

### DISCUSSION THREAD

**Discussion by Akalanka1337** [community/community#18401] [Other Feature Feedback, Questions, & Ideas]
Its better to have a privacy options like hide starred repositories from public view.

**Comment by miteokaku** [comment]
👌

**Comment by NitinIsFit** [comment]
True, i agree with this

**Comment by olafkotur** [comment]
Just trying to understand this a little better - why may you want to hide your starred repos in the first place? 🤔

**Reply by NitinIsFit** [reply]
Because my friend said "bro star my repo" an i doh want to, then i starred an another repo made by someone else, and my friend said "bro you starred his repo and didn't star mine" so I wanna hide every of them💀

**Reply by NitinIsFit** [reply]
And there are some embarrassing repos i like but I wanna hide like "anime girls holding programming books", "Jojo characters holding programming books"?

**Reply by Schecher1** [reply]
"bro star my repo" I can absolutely understand you. For this, the function would be just "awesome".

**Reply by NitinIsFit** [reply]
Arent we allowed to edit messages in threads?
